app/src/main/java/org/jellyfin/androidtv/ui/home/HomeFragmentLiveTVRow.kt

```diff
@@ -0,0 +1,76 @@
+package org.jellyfin.androidtv.ui.home
+
+import android.app.Activity
+import android.content.Intent
+
+import androidx.leanback.widget.ArrayObjectAdapter
+import androidx.leanback.widget.HeaderItem
+import androidx.leanback.widget.ListRow
+import androidx.leanback.widget.OnItemViewClickedListener
+import androidx.leanback.widget.Presenter
+import androidx.leanback.widget.Row
+import androidx.leanback.widget.RowPresenter
+
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.TvApp
+import org.jellyfin.androidtv.browsing.BrowseRecordingsActivity
+import org.jellyfin.androidtv.browsing.BrowseScheduleActivity
+import org.jellyfin.androidtv.browsing.UserViewActivity
+import org.jellyfin.androidtv.livetv.LiveTvGuideActivity
+import org.jellyfin.androidtv.presentation.CardPresenter
+import org.jellyfin.androidtv.presentation.GridButtonPresenter
+import org.jellyfin.androidtv.ui.GridButton
+import org.jellyfin.apiclient.model.dto.BaseItemDto
+
+class HomeFragmentLiveTVRow(val activity: Activity?) : HomeFragmentRow(), OnItemViewClickedListener {
+	override fun addToRowsAdapter(cardPresenter: CardPresenter?, rowsAdapter: ArrayObjectAdapter?) {
```

Comment:
When overriding Java methods in Kotlin you can actually choose if an parameter is non-null. Removing the question marks from the functions parameters so you don't need to use the non-null operator (`!!`) in the implementation.

_Same for some other places in this file_